From 68542578ced19ecf132a95d219eb51cb6464dd36 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Wed, 12 Feb 2025 22:01:58 -0500
Subject: [PATCH] Replace pytest-runner with pytest for testing pyp2rpm

This does not change the spec files generated by pyp2rpm, but it gets
"tox" working again with current setuptools releases.

The `pytest-runner` package has been deprecated upstream for some time,
and the project is now archived:
https://github.com/pytest-dev/pytest-runner/tree/v6.0.1?tab=readme-ov-file#deprecation-notice

Additionally, the long-deprecated "setup.py test" command was removed in
setuptools 72:
https://github.com/pypa/setuptools/blob/v75.8.0/NEWS.rst#v7200
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 321d8f7..e801d77 100644
--- a/setup.py
+++ b/setup.py
@@ -53,7 +53,6 @@ setup(
                       ],
     setup_requires=['setuptools',
                     'flexmock >= 0.9.3',
-                    'pytest-runner',
                     'click',
                     'Jinja2',
                     ],
-- 
2.48.1

